modules/simphera_aws_instance/postgresql.tf

```diff
@@ -1,37 +1,38 @@
 resource "aws_db_instance" "simphera" {
-  allocated_storage      = var.postgresqlStorage
-  max_allocated_storage  = var.postgresqlMaxStorage
-  engine                 = "postgres"
-  engine_version         = var.postgresqlVersion
-  instance_class         = var.db_instance_type_simphera
-  identifier             = "${local.instancename}-simphera"
-  db_name                = replace("${local.instancename}simphera", "/[^0-9a-zA-Z]/", "") # Use alphanumeric characters only
-  username               = local.secret_postgres_username
-  password               = local.secrets["postgresql_password"]
-  skip_final_snapshot    = true
-  db_subnet_group_name   = "${var.infrastructurename}-vpc"
-  vpc_security_group_ids = [var.postgresql_security_group_id]
-  apply_immediately      = true
-  tags                   = var.tags
-
+  allocated_storage       = var.postgresqlStorage
+  max_allocated_storage   = var.postgresqlMaxStorage
+  engine                  = "postgres"
+  engine_version          = var.postgresqlVersion
+  instance_class          = var.db_instance_type_simphera
+  identifier              = "${local.instancename}-simphera"
+  db_name                 = replace("${local.instancename}simphera", "/[^0-9a-zA-Z]/", "") # Use alphanumeric characters only
+  username                = local.secret_postgres_username
+  password                = local.secrets["postgresql_password"]
+  skip_final_snapshot     = true
+  db_subnet_group_name    = "${var.infrastructurename}-vpc"
+  vpc_security_group_ids  = [var.postgresql_security_group_id]
+  apply_immediately       = true
+  tags                    = var.tags
+  backup_retention_period = var.db_retention_simphera

```

Comment:
Please note that the ref architecture is going to support continuous backups.

Reply:
It is doing so by default. The used variable is coming with the default 7 days. I added the config to support usage in development environments and testing.